jekyll/_cci2/release/set-up-a-release-environment.adoc

```diff
@@ -0,0 +1,111 @@
+
+* Restore a version
+* Scale a version
+* Restart a version
+* Retry a release (when using Argo Rollouts)
+* Promote a release (when using Argo Rollouts)
+* Cancel a release (when using Argo Rollouts)
+
+[#prerequisites]
+== Prerequisites
+
+In order to set up a release environment with CircleCI you will need to ensure the following prerequisites are fulfilled:
+
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
+* A Kubernetes cluster
+* A project building on CircleCI that deploys to your Kubernetes cluster. See the xref:../create-project#[Create a project] page for steps.
+* You must have write access to the project associated with the component being deployed. For full details of roles and permissions for GitLab and GitHub App integrations, see the xref:../roles-and-permissions-overview.adoc#[Roles and permissions overview]. If your integration is through Bitbucket or the GitHub OAuth app, you will need to be an org admin in Bitbucket/GitHub respectively. To find out which GitHub integration you have, see the xref:../github-apps-integration.adoc#[GitHub App integration] page
+
+CircleCI supports Kubernetes, Helm, and Argo Rollouts. The supported versions are listed below:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+|System
+|Supported versions
+
+|Kubernetes
+a| 1.28.0 +
+1.27.0
+
+|Helm
+a| 3.12.0 +
+3.11.0
+
+|Argo Rollouts (optional)
+a| 1.6.0 +
+1.5.0
+|===
+
+[#create-release-environment]
+== 1. Create your release environment and token
+
+. In the CircleCI web app, click **Releases** in the sidebar
+. Click btn:[Add Release Environment]
+. Enter a name for your release environment and click btn:[Create Environment]
+. Click to select the environment you just created
+. Click btn:[Create New Token] and click btn:[Create Token]
+. The new token will be displayed. Be sure to copy and save this somewhere for use in the next step. Then click btn:[Done]
+
+[#add-release-agent]
+== 2. Add the release agent into your Kubernetes cluster
+
+Use Helm to install the CircleCI release agent into your Kubernetes cluster
+
+. Add the link:https://circleci-public.github.io/cci-k8s-release-agent/[CircleCI release agent Helm repository] by running the following command:
++
+[,shell]
+----
+helm repo add release-agent https://circleci-public.github.io/cci-k8s-release-agent
+----
+
+. Then, update your local Helm chart repository cache
++
+[,shell]
+----
+helm repo update
+----
+
+[#install-the-release-agent]
+== 3. Install the release agent
+
+Install the Helm chart to integrate CircleCI's release agent into your Kubernetes cluster. Be sure to substitute the token you created in the previous step, and also include a list of Kubernetes namespaces you want to manage:
+
+[,shell]
+----
+helm install circleci-release-agent-system release-agent/circleci-release-agent \
```

Comment:
we should switch this to `upgrade --install` that way its relevant for updating and installing.

Reply:
I have it on my list to create some upgrade instructions so maybe that can be separate?

Reply:
I think I'll handle upgrade installs in a separate PR after this initial set of docs is live cc @ryanbarkley if you think otherwise, or if I should update to @meeech's suggestion just let me know

Reply:
OK I'll change to @meeech's suggestion